compositor_common/src/scene/builtin_transformations.rs

```diff
@@ -56,6 +58,8 @@ pub enum TransformToResolutionStrategy {
     Fit {
         #[serde(default)]
         background_color_rgba: RGBAColor,
+        horizontal_align: HorizontalAlign,
+        vertical_align: VerticalAlign,
```

Comment:
Question to reviewers: any better names?
I thought about using the API most compliant with CSS, but after analyzing how unreadable this is in CSS, I left it as it is.

Reply:
I think this is fine, but I would change '\*align\*' -> '\*alignment\*'

Reply:
In text renderer we use `align` to be consistent with CSS `text-align`, so we have to decide to either:
1. Use everywhere one - `align` or `alignment`
2. Use both `align` and `alignment`, picking the one most suitable / sounding better to a specific case

I have no preference, but if this changes your decision / doesn't seem right to you, let me know. Otherwise, I will use `alignment` here and keep `align` in the text renderer.